readme.md

```diff
@@ -105,11 +105,11 @@
 - [RubyMotion](https://github.com/motion-open-source/awesome-rubymotion#readme) - Write cross-platform native apps for iOS, Android, macOS, tvOS, and watchOS in Ruby.
 - [Smart TV](https://github.com/vitalets/awesome-smart-tv#readme) - Create apps for different TV platforms.
 - [GNOME](https://github.com/Kazhnuz/awesome-gnome#readme) - Simple and distraction-free desktop environment for Linux.
+- [KDE](https://github.com/francoism90/awesome-kde#readme) - Plasma is KDE's flagship product, offering a customizable desktop environment.
```

Comment:
The list should be named `KDE Plasma` since it's about that specifically.

The description could be:

> Customizable Linux desktop environment.

Reply:
@sindresorhus Will change this into KDE, as it also mentions other stuff as well.

Reply:
I have updated and fixed the linting errors. It may be required to create a new PR, please let me know if it's ready to merge.